### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by shdwy** [hop-protocol/hop-airdrop#366]
# Related Addresses

```
0x37ccb985aa7de1af158221b7af17a7c50d34ff50
0x287a9a9ed1442291f9ce5e2a9a6f8e6472c1ba24
0x4a85c7f7e36a5d7ee6fc027a454b64808a2588c4
0xd26995334aed95ffff9f41757bdc72606717e021
0xaa6d9432fd9a935cd6fcbd8cd1117c0ea279a9cd
0x10bd341ce4c5e0c9fd5b7903eeff88f5773d5a7a
0x4bb555e1da9b2699729efa291455ca7807ea39bb
0x7c32be4221bfd3eb626a4da5906904eed6467d57
0x1d4a325985ff23c7eab5b256eef960243162fbac
0x8426796713624b06d78c7647413618a7efdc4391
0x3f14cb3f3e557261d055e7815860863f8022aa56
0x57b21989eca6557736cf6f9ae23b72548f82f46f
0x86c2e58b86693816d805d7daa569142a44b1bf33

Started hopping here.
0x301580d7b1be85bf4a546bfbc4357c0de2df6030
0xb0e17d13c5100f353a0867b03afe6b000db1b0b2
0xdcb1a6641d788b12d20120197e242fe05b12daf3
0xb7bb497992ff845670cf2d819ffc1feff4e26038
0x76846ffea632b0757fafa9f108cb29846f314e31
0x9b466d81d216a2648b8e7e1e0b70d05d40bf826a
0x11ea0dedde50ffb19e43bd88eda907965cea1038
0x402f5fc06098043ff1bd027b34bfdb70653092c6
0x994cc97637808e7ccf15fb50d8ff0ec76f5facdc
0xbb66b602add0544c833415b74973d8512affaa9a
0xe471cbd11444e861f18256e917de28746c5c8fb6
0x6a2d9e0ae404c6b40b06a8e92ea1190ba807568e
0xb74627704ebae9c57b221ef3a9a550e5dbc5c038
0xf575deba8bce84edad9902dfed3f32ed23854fdb
0x2732c0f06220619b884484125ab3c613cadba741
0x9dba28601f15043023920fb7ea5f7c7cd8a245f4
0x457d3eb4d849afbab1631c2792495c8443c9b38b
0x589aaf94c2cf5ec770f88dc8a1ba0dda315e190d
0xab8bbd133757d7f2d59f3b1ad20e407aeb40b7f6
0x28cf32b0603d7855851a55d02b56934762f5f2d4
0x684c4db713f35ef19836ce28c9faf35440aea6aa

```

# Reasoning

All addresses interact with Hop, then transfer gas to the next address on Gnosis Chain sequentially. The timestamps of these transactions are incremental and the token amounts transferred decrease a little on each tx. After address 0x86c2e58b86693816d805d7daa569142a44b1bf33, the attacker started adding a "hop" between each attack, transfering funds to a wallet with no activity, then transferring the funds to the next address on the list.

# Methodology

I used blockscout API to figure out which address originally funded all 30k address eligible for the airdrop. Then I used excel to try and find chains of addresses that funded each other. Once I found a chain, I manually checked their activity on blockscout to analyze their behaviour and follow their trail.

# Rewards Address

0xD47CB1835EB36e4A108C33E446449192426Fc160

**Comment by Firsttimer001** [comment]
Karma is a bitch. 
There is a God watching.
Snitches get stitches, disabled kids, and never-ending hell fire.

This was my last chance to get back to crypto debtless which I prepared months in advance, I guess you get paid to snitch. Karma is a bitch, enjoy your short term happiness :).

**Comment by shdwy** [comment]
I'm guessing these were your addresses? Should've taken the 25% and self reported mate.

**Comment by tryhard3eth** [comment]
> 0x589aaf94c2cf5ec770f88dc8a1ba0dda315e190d

you should have waited with this reply, your wallets are already known and mentioned in the "groups.json", but the team decided to not eliminate you.

**Comment by shanefontaine** [comment]
Thank you for your report @shdwy. We have verified that the addresses in this report are Sybil attackers.

The report included 21 eligible addresses as Sybil attackers which means you are eligible for 3252.189489645225050938 HOP! When Hop DAO is live, we will make a proposal for this reward — subject to a 1 year lockup, as mentioned in the [original Mirror post](https://hop.mirror.xyz/AI5fOUR0X_l0mktShDOx3mwr-hsB24gp8GvTWtS-MBc).

It looks like some of the addresses you reported were successfully reported in #242.

The qualified addresses are as follows:

```
0x301580d7b1be85bf4a546bfbc4357c0de2df6030
0xb0e17d13c5100f353a0867b03afe6b000db1b0b2
0xdcb1a6641d788b12d20120197e242fe05b12daf3
0xb7bb497992ff845670cf2d819ffc1feff4e26038
0x76846ffea632b0757fafa9f108cb29846f314e31
0x9b466d81d216a2648b8e7e1e0b70d05d40bf826a
0x11ea0dedde50ffb19e43bd88eda907965cea1038
0x402f5fc06098043ff1bd027b34bfdb70653092c6
0x994cc97637808e7ccf15fb50d8ff0ec76f5facdc
0xbb66b602add0544c833415b74973d8512affaa9a
0xe471cbd11444e861f18256e917de28746c5c8fb6
0x6a2d9e0ae404c6b40b06a8e92ea1190ba807568e
0xb74627704ebae9c57b221ef3a9a550e5dbc5c038
0xf575deba8bce84edad9902dfed3f32ed23854fdb
0x2732c0f06220619b884484125ab3c613cadba741
0x9dba28601f15043023920fb7ea5f7c7cd8a245f4
0x457d3eb4d849afbab1631c2792495c8443c9b38b
0x589aaf94c2cf5ec770f88dc8a1ba0dda315e190d
0xab8bbd133757d7f2d59f3b1ad20e407aeb40b7f6
0x28cf32b0603d7855851a55d02b56934762f5f2d4
0x684c4db713f35ef19836ce28c9faf35440aea6aa
```

**Comment by shanefontaine** [comment]
It looks like all addresses had similar behaviors on the Hop protocol as well:


address | ensName | total | mainnet | arbitrum | optimism | polygon | xdai | totalVolume
-- | -- | -- | -- | -- | -- | -- | -- | --
0x457d3eb4d849afbab1631c2792495c8443c9b38b |   | 14 | 0 | 0 | 0 | 7 | 7 | $2,184.87
0x589aaf94c2cf5ec770f88dc8a1ba0dda315e190d |   | 14 | 0 | 0 | 0 | 7 | 7 | $2,165.31
0x28cf32b0603d7855851a55d02b56934762f5f2d4 |   | 14 | 0 | 0 | 0 | 7 | 7 | $2,091.62
0x6a2d9e0ae404c6b40b06a8e92ea1190ba807568e |   | 13 | 0 | 0 | 0 | 6 | 7 | $2,073.41
0x684c4db713f35ef19836ce28c9faf35440aea6aa |   | 14 | 0 | 0 | 0 | 7 | 7 | $2,049.75
0xf575deba8bce84edad9902dfed3f32ed23854fdb |   | 15 | 0 | 0 | 0 | 8 | 7 | $2,036.94
0xb74627704ebae9c57b221ef3a9a550e5dbc5c038 |   | 13 | 0 | 0 | 0 | 6 | 7 | $2,019.82
0xab8bbd133757d7f2d59f3b1ad20e407aeb40b7f6 |   | 12 | 0 | 0 | 0 | 6 | 6 | $2,019.00
0xb0e17d13c5100f353a0867b03afe6b000db1b0b2 |   | 2 | 0 | 0 | 0 | 1 | 1 | $1,990.30
0xdcb1a6641d788b12d20120197e242fe05b12daf3 |   | 2 | 0 | 0 | 0 | 1 | 1 | $1,990.08
0x301580d7b1be85bf4a546bfbc4357c0de2df6030 |   | 2 | 0 | 0 | 0 | 1 | 1 | $1,989.96
0x2732c0f06220619b884484125ab3c613cadba741 |   | 13 | 0 | 0 | 0 | 6 | 7 | $1,912.97
0x9dba28601f15043023920fb7ea5f7c7cd8a245f4 |   | 14 | 0 | 0 | 0 | 7 | 7 | $1,912.65
0xbb66b602add0544c833415b74973d8512affaa9a |   | 14 | 0 | 0 | 0 | 7 | 7 | $1,859.79
0x402f5fc06098043ff1bd027b34bfdb70653092c6 |   | 13 | 0 | 0 | 0 | 6 | 7 | $1,821.72
0x76846ffea632b0757fafa9f108cb29846f314e31 |   | 4 | 0 | 0 | 0 | 2 | 2 | $1,774.68
0x11ea0dedde50ffb19e43bd88eda907965cea1038 |   | 11 | 0 | 0 | 0 | 5 | 6 | $1,729.99
0x994cc97637808e7ccf15fb50d8ff0ec76f5facdc |   | 13 | 0 | 0 | 0 | 6 | 7 | $1,704.92
0x9b466d81d216a2648b8e7e1e0b70d05d40bf826a |   | 4 | 0 | 0 | 0 | 2 | 2 | $1,664.80
0xe471cbd11444e861f18256e917de28746c5c8fb6 |   | 9 | 0 | 0 | 0 | 7 | 2 | $1,634.16
0xb7bb497992ff845670cf2d819ffc1feff4e26038 |   | 4 | 0 | 0 | 0 | 2 | 2 | $1,627.12

**Comment by 666erxia** [comment]
Family members die every day, and it is not easy to die 2919.738533629433120345 HOP! buy you a coffin

**Comment by forweb3** [comment]
sir ，One of my, our report is finding that the rule has to be a jump to survive. However, the successor has completed more than 10 user tracking records, and there is no record of him. I cancelled his reward. And this is what I found so am I not rewarded accordingly? This address has not interacted with hop. And he did not provide other information that he interacted. I report all his reports.  
0xD47CB1835EB36e4A108C33E446449192426Fc160 This is a brand new empty address, why can't he provide the address he interacts with, because he is the real empty hunter. He circumvented it with better technical isolation. So here's my reasoning, I want 25% of his reward to go to me and then the remaining 75% to all eligible real users. Just like our rules.
@shanefontaine
